admin/customers.php

```diff
@@ -1057,7 +1057,7 @@ function check_form() {
 
                   if (!empty($cInfo->lifetime_value)) {
                     $contents[] = array('text' => TEXT_INFO_LIFETIME_VALUE . ' ' . $currencies->format($cInfo->lifetime_value));
-                    $contents[] = array('text' => TEXT_INFO_LAST_ORDER . ' ' . zen_date_short($cInfo->last_order['date_purchased']) . '<br>' . TEXT_INFO_ORDERS_TOTAL . ' ' . $currencies->format($cInfo->last_order['order_total'], true, $cInfo->last_order['currency'], $cInfo->last_order['currency_value']));
+                    $contents[] = array('text' => TEXT_INFO_LAST_ORDER . ' ' . zen_date_short($cInfo->last_order['date_purchased']) . '<br>' . TEXT_INFO_ORDERS_TOTAL . ' ' . (float)$currencies->format($cInfo->last_order['order_total'], true, $cInfo->last_order['currency'], (float)$cInfo->last_order['currency_value']));
```

Comment:
It appears the first float added is in the wrong place.

Reply:
Yes it was; but as noted for #4870, that line in `customers.php` isn't going to fly anyway.  I'll resubmit to correct both #4870 and #4871 in one-swell-foop.